runtime/oti/j9consts.h

```diff
@@ -447,6 +447,12 @@ extern "C" {
 #define J9_FINDCLASS_FLAG_NO_ISOLATION 0x2000
 #define J9_FINDCLASS_FLAG_ANON 0x4000
 #define J9_FINDCLASS_FLAG_FIND_MODULE_ON_FAIL 0x8000
+/*
+ * Used to permit a caller to find classes that are frozen and are originally
+ * loaded in defineClassesCommon.
+ */
+#define J9_FINDCLASS_FLAG_JCL_DEFINE_CLASS 0x10000
```

Comment:
this should be defined as 0x1000000 and moved after `J9_FINDCLASS_FLAG_CLASS_OPTION_NULL_RESTRICTED_ARRAY`